src/main/java/sleepy/Sleepy.java

```diff
@@ -0,0 +1,58 @@
+package sleepy;
+
+import sleepy.taskstorage.TaskList;
+import sleepy.tools.Parser;
+import sleepy.tools.ResponseHandler;
+
+/**
+ * This class is the main class for the Sleepy AI Chatbot.
+ *
+ * @author kjw142857
+ */
+public class Sleepy {
+    public static final String WELCOME_TEXT = "Hello! I'm Sleepy \n Zzz...";
+    private TaskList taskList;
+
+    private boolean isInExitState;
+
+    /**
+     * Constructor for the Sleepy class.
+     */
+    public Sleepy() {
+        String filePath = "./storage/HardDiskStorage.txt";
+        // Retrieve saved data
+        taskList = new TaskList(filePath);
+        isInExitState = false;
+    }
+
```

Comment:
Consider to [specify file paths in an OS-independent way](https://www.sghill.net/how-do-i-make-cross-platform-file-paths-in-java.html).

Reply:
I never knew about that, thank you for enlightening me on this! Will seriously consider implementing this to support different OSes.